_sample_code/osgi_codings_tasks/bundles/org.openhab.training.helloosgi.modified/about.html

```diff
@@ -0,0 +1,29 @@
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
+    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"/>
+<title>About</title>
+</head>
+<body lang="EN-US">
+<h2>About This Content</h2>
+
+<p>&lt;<em>September 15, 2014</em>&gt;</p>
+<h3>License</h3>
+
+<p>The Eclipse Foundation makes available all content in this plug-in (&quot;Content&quot;).  Unless otherwise
```

Comment:
Reading this, it reminds me that we need to rework this text - "The Eclipse Foundation" is not involved in any way here... What about saying "The openHAB community"?